INSTALLATION.md

```diff
@@ -49,7 +49,13 @@ Simply use the link below to download and then open the file on your machine. Th
 Once the installation is finished, congratulations, you can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](http://flappy.massa) can help.
 If you want to get down to business, you can start your [Massalian journey](http://my.massa/thyra/wallet) right away!
 
-Download and open the file [here](https://github.com/massalabs/thyra/blob/main/scripts/macos_install.sh).
+Download and open the file [here](https://github.com/massalabs/thyra/blob/main/scripts/macos_install.sh) or or run the following cmd:
+
+```sh
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/massalabs/thyra/main/scripts/macos_install.sh)"
+```
+
+Congratulation, your version of Thyra is now installed on your system and can be run by executing `./thyra-server` in your terminal.
```

Comment:
This is wrong the binary is located in `/usr/local/bin` you can't launch it with `./` either you have `/usr/local/bin` in the your `PATH` environment variable and you just do `thyra-server` otherwise you have to specify the full path : `/usr/local/bin/thyra-server`.

The `./` you have written is for "current directory" but `thyra-server` isn't in the current directory when downloading with the script.

I think you can put only `thyra-server` because having `/usr/local/bin` in your `PATH` environment variable is common on all MacOS by default. If someone modified it then he surely know how to find the full path himself.

Reply:
thanks a lot! it's updated